server/com.eclipsesource.uml.modelserver/src/main/java/com/eclipsesource/uml/modelserver/commands/RemoveAssociationCommand.java

```diff
@@ -0,0 +1,24 @@
+/********************************************************************************
+ * Copyright (c) 2021 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0, or the MIT License which is
+ * available at https://opensource.org/licenses/MIT.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR MIT
+ ********************************************************************************/
+package com.eclipsesource.uml.modelserver.commands;
+
+import org.eclipse.emf.common.command.CompoundCommand;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.edit.domain.EditingDomain;
+
+public class RemoveAssociationCommand extends CompoundCommand {
+
+   public RemoveAssociationCommand(final EditingDomain domain, final URI modelUri, final String semanticUriFragment) {
+      this.append(new RemoveAssociationElementCommand(domain, modelUri, semanticUriFragment));
```

Comment:
This assumes that Association Ends (Properties) are always owned by the Association; which is true for Associations created by "AddAssociationElementCommand", but isn't true for all UML Associations: UML Association ends can be either "Owned by Classifier" or "Owned by Association". We should make sure that all Association Ends are deleted as well

Reply:
Ah that's a good hint, thanks! As we are not able to create other Association Ends at the moment, I added a `TODO` comment to this command, such that any adopters are aware of removing the additional usages.